src/masternodes/rpc_icxorderbook.cpp

```diff
@@ -94,7 +126,7 @@ UniValue icxcreateorder(const JSONRPCRequest& request) {
                         {
                             {"tokenFrom|chainFrom", RPCArg::Type::STR, RPCArg::Optional::OMITTED, "Symbol or id of selling token/chain"},
```

Comment:
I don't see reason to store chain name, it does not store any kind of value.

Reply:
I have introduced chain name as I made this not only for BTC, but it can be any BTC protocol chain.
If we are going to do only BTC and it will be necessary to make hardfork in order to introduce new chain then I can remove chain name and bind it only to BTC!